DecaTec.WebDav/AbsoluteUri.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+
+namespace DecaTec.WebDav
+{
+    /// <summary>
+    /// Represents the Absolute-URI object as specified in <see href="https://tools.ietf.org/html/rfc3986#section-4.3"/>.
+    /// </summary>
+    public class AbsoluteUri
+    {
+        /// <summary>
+        /// The URI of this <see cref="AbsoluteUri"/>.
+        /// </summary>
+        private readonly Uri absoluteUri;
+
+        private AbsoluteUri(Uri absoluteUri)
```

Comment:
Add a comments describes the factory method must be used and create a `<see/>` link to it.